README.md

```diff
@@ -122,19 +122,16 @@ and converts it to parquet format, leveraging the custom loader code specified i
 you must manually re-run the ingest step in order to use the updated dataset in the recipe. 
 The ingest step does not automatically detect changes in the dataset.
 
-The custom loader function allows use of datasets in other formats, such as `csv`. 
-The function should be defined in [`steps/ingest.py`](https://github.com/mlflow/recipes-regression-template/blob/main/steps/ingest.py),
-and should accept two parameters:
-- `file_path`: `str`. Path to the dataset file.
-- `file_format`: `str`. The file format string, such as `"csv"`.
-
-It should return a Pandas DataFrame representing the content of the specified file. [`steps/ingest.py`](https://github.com/mlflow/recipes-regression-template/blob/main/steps/ingest.py) contains an example placeholder function.
+Below are all the possible options and full reference guide for different configurations allowed in ingest step:
 
 The input dataset is specified by the `steps.ingest` section in [`recipe.yaml`](https://github.com/mlflow/recipes-regression-template/blob/main/recipe.yaml) as follows: 
 <details>
 <summary><strong><u>Full configuration reference</u></strong></summary>
```

Comment:
We don't need two level nesting. Can we directly show the 4 `using` sections with each folded by default?